### PR TITLE
Feat: Explore page with post

### PR DESCRIPTION
**Pull request by pritam-kr**
- [Feat: Explore page with post](https://github.com/pritam-kr/like/pull/11/commits/09742a66d67e2dc93dc44e4f7bc8c92b3b9cf159)
      - Post like
      - navigate to Comment page
      - Loading
      - [Feat: Signup feature](https://github.com/pritam-kr/like/pull/11/commits/a5951749475964c8a958217dc82db33d2355ced7)
      - [Feat: logout Handler](https://github.com/pritam-kr/like/pull/11/commits/2437b334e258967c7b67366ec22e4babbefc577a)
      - [Feat: user profile](https://github.com/pritam-kr/like/pull/11/commits/9f7c02072c69ab311ce58fa0fdcf2e60debfc55a)
      - [Feat: User profile edit feature](https://github.com/pritam-kr/like/pull/11/commits/dc99f23e273d177bdbaf7561a77bb2a0d1ca5689)

### CODE REVIEW

**Review by Chakravarthi0 — APPROVED** [review]
the comments make the code easy to understand
clean code
good to merge

File: src/Pages/Explore/Explore.jsx [hunk 1]
```diff
@@ -1,12 +1,168 @@
...
+//   <div className="flex justify-center items-center">
+//     <span className="mr-2">
+//       <FaIcons.FaTrash
+//         className="icons profile-icons"
+//         // onClick={() => postDeleteHandler(_id, token)}
+//       />
+//     </span>
+//     <span>
+//       <FaIcons.FaEdit
+//         className="icons profile-icons"
```
Comment (gopalbbharadva) [inline src/Pages/Explore/Explore.jsx:133]: Remove these comments if not needed.

File: src/Components/Topbar/Topbar.jsx [hunk 2]
```diff
@@ -4,9 +4,16 @@ import * as AiIcons from "react-icons/ai";
...
+import { logoutHandler } from "../../Store/Slice/AuthSlice";
+import { useDispatch } from "react-redux";
 
 const Topbar = () => {
   const { setPostModal } = useModalContext();
+   const dispatch = useDispatch()
+
+   const logoutUserHandler = () => {
+    dispatch(logoutHandler())
+   }
```
Comment (gopalbbharadva) [inline src/Components/Topbar/Topbar.jsx:16]: it can be write in single line

**Review by gopalbbharadva — APPROVED** [review]
Functionality working fine, Signup validation also works fine. Good to Go.

File: src/Pages/Explore/Explore.jsx (hunk 1 above)
Comment (pritam-kr) [reply src/Pages/Explore/Explore.jsx:133]: Done

File: src/Components/Topbar/Topbar.jsx (hunk 2 above)
Comment (pritam-kr) [reply src/Components/Topbar/Topbar.jsx:16]: ok

Comment (pritam-kr) [reply src/Components/Topbar/Topbar.jsx:16]: ok

**Review by manojasarsa — APPROVED** [review]
The features are working as expected. GTG!

**Pull request merged**